### ISSUE TITLE
Nested groups does not work

### ISSUE THREAD

**Issue by MarcinOrlowski** [skipperbent/simple-php-router#83]
```
SimpleRouter::group(['prefix' => 'v1'], function () {
	SimpleRouter::get('/', function () {
		echo "OK 1";
	});
	SimpleRouter::group(['prefix' => '/system'], function () {
		SimpleRouter::get('/', function () {
			echo "OK 2";
		});
	});
});
```

Route `/v1` works, but `/v1/system` does not.

**Comment by skipperbent** [comment]
This bug seems only to affect `/` urls in nested groups. Try to change the nested groups route url to `/something-else`.

Theres a fix for this bug in the next release.

**Comment by skipperbent** [comment]
This is fixed in version 1.7.

I've also added a test for nested groups.
